Common/src/main/resources/assets/livingthings/lang/zh_cn.json

```diff
@@ -0,0 +1,115 @@
+{
+
+  "itemGroup.livingthings.general":"鲜活生命",
```

Comment:
Can you reverse this line to `Living Things`, I like to keep the item-group name consistent between all languages